lib/driver/aws-sigv4-driver-remote-connection.js

```diff
@@ -68,6 +67,7 @@ class AwsSigV4DriverRemoteConnection extends gremlin.driver.RemoteConnection {
     const { url, headers } = getUrlAndHeaders(this.host, this.port, this.options, '/gremlin', this.secure ? 'wss' : 'ws');
     debug(`connect: ${JSON.stringify(url, headers)} (try #${this.try})`);
     this._client = new gremlin.driver.Client(url, ({ headers, ...this.clientOptions }));
+    this._client.open();
```

Comment:
`.open()` returns a promise, this should be in await. The method should become an async method due to that.